Plugin fails to compile due to path issues

Compilation currently fails with the following error message:

> scons: Reading SConscript files ...
Building for platform "linuxbsd", architecture "x86_64", target "editor".
This file `./systems/system_structs.gen.h` is already generated.
This file `./systems/system_exe_funcs.gen.h` is already generated.
This file `./systems/system_dispatcher_exe_funcs.gen.h` is already generated.
This file `./systems/temporary_system_exe_funcs.gen.h` is already generated.
Checking for C header file mntent.h... (cached) yes
scons: done reading SConscript files.
scons: Building targets ...
[Initial build] Compiling tests/test_main.cpp ...
[Initial build] Compiling modules/voxel/editor/terrain/editor_property_aabb_min_max.cpp ...
[Initial build] Compiling modules/voxel/editor/terrain/voxel_terrain_editor_inspector_plugin.cpp ...
In file included from modules/voxel/editor/terrain/editor_property_aabb_min_max.h:6,
                 from modules/voxel/editor/terrain/editor_property_aabb_min_max.cpp:1:
modules/voxel/editor/terrain/../../util/godot/classes/editor_spin_slider.h:5:10: fatal error: editor/editor_spin_slider.h: No such file or directory
    5 | #include <editor/editor_spin_slider.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
scons: *** [modules/voxel/editor/terrain/editor_property_aabb_min_max.linuxbsd.editor.x86_64.o] Error 1
In file included from modules/voxel/editor/terrain/editor_property_aabb_min_max.h:6,
                 from modules/voxel/editor/terrain/voxel_terrain_editor_inspector_plugin.cpp:5:
modules/voxel/editor/terrain/../../util/godot/classes/editor_spin_slider.h:5:10: fatal error: editor/editor_spin_slider.h: No such file or directory
    5 | #include <editor/editor_spin_slider.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
scons: *** [modules/voxel/editor/terrain/voxel_terrain_editor_inspector_plugin.linuxbsd.editor.x86_64.o] Error 1
^Cscons: *** [tests/test_main.linuxbsd.editor.x86_64.o] Build interrupted.
scons: Build interrupted.
scons: building terminated because of errors.
scons: writing .sconsign file.
[Time elapsed: 00:00:22.379]

I suspect the compilation instructions need updating as the github workflow seems to build.

--- a/util/godot/classes/editor_file_dialog.h
+++ b/util/godot/classes/editor_file_dialog.h
@@ -2,7 +2,7 @@
 #define ZN_GODOT_EDITOR_FILE_DIALOG_H
 
 #if defined(ZN_GODOT)
-#include <editor/editor_file_dialog.h>
+#include <editor/gui/editor_file_dialog.h>
 #elif defined(ZN_GODOT_EXTENSION)
 #include <godot_cpp/classes/editor_file_dialog.hpp>
 using namespace godot;

--- a/util/godot/classes/editor_spin_slider.h
+++ b/util/godot/classes/editor_spin_slider.h
@@ -2,7 +2,7 @@
 #define ZN_GODOT_EDITOR_SPIN_SLIDER_H
 
 #if defined(ZN_GODOT)
-#include <editor/editor_spin_slider.h>
+#include <editor/gui/editor_spin_slider.h>
 #elif defined(ZN_GODOT_EXTENSION)
 #include <godot_cpp/classes/editor_spin_slider.hpp>
 using namespace godot;

